src/libwolfboot.c

```diff
@@ -974,20 +974,50 @@ int RAMFUNCTION wolfBoot_set_encrypt_key(const uint8_t *key,
     return 0;
 }
 
+int RAMFUNCTION wolfBoot_backup_encrypt_key(const uint8_t* key,
+    const uint8_t* nonce)
+{
+#ifndef MMU
+    uint32_t magic[2] = {WOLFBOOT_MAGIC, WOLFBOOT_MAGIC_TRAIL};
+
+    XMEMCPY((uint8_t*)WOLFBOOT_PARTITION_BOOT_ADDRESS, key, ENCRYPT_KEY_SIZE);
```

Comment:
not sure how this would work, XMEMCPY straight to NVM? Did you mean to use `hal_flash_write`?

Reply:
whoops, I've got simulator brain. thought you could read and write that way